lib/diagnostic.js

```diff
@@ -5,10 +5,30 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
+
+
+
+// NOTE: I am unable to do this diagnostic as I still do not understand callbacks. I have requested TA assistence
+//week and will be meeting with Jordan to review this topic. I haven't had
+//any free time yet to review at home online which i plan to do over weekend.
```

Comment:
No worries Conor, a lot of people had issues with this particular topic so we will be doing a review during workshop today